doc/man3/OSSL_CMP_CTX_new.pod

```diff
@@ -497,36 +502,36 @@ sent to remote.
 
 OSSL_CMP_CTX_set0_newPkey() can be used to explicitly set the given EVP_PKEY
 structure as the private or public key to be certified in the CMP context.
-The B<priv> parameter must be 0 if and only if the given key is a public key.
+The I<priv> parameter must be 0 if and only if the given key is a public key.
 
 OSSL_CMP_CTX_get0_newPkey() gives the key to use for certificate enrollment
 dependent on fields of the CMP context structure:
 the newPkey (which may be a private or public key) if present,
 else the public key in the p10CSR if present, else the client private key.
-If the B<priv> parameter is not 0 and the selected key does not have a
+If the I<priv> parameter is not 0 and the selected key does not have a
 private component then NULL is returned.
 
 OSSL_CMP_CTX_set1_issuer() sets the name of the intended issuer that
 will be set in the CertTemplate, i.e., the X509 name of the CA server.
 
 OSSL_CMP_CTX_set1_subjectName() sets the subject DN that will be used in
 the CertTemplate structure when requesting a new cert. For Key Update Requests
-(KUR), it defaults to the subject DN of the B<reference certificate>,
-see B<OSSL_CMP_CTX_set1_oldCert()>. This default is used for Initialization
+(KUR), it defaults to the subject DN of the reference certificate,
+see OSSL_CMP_CTX_set1_oldCert(). This default is used for Initialization
 Requests (IR) and Certification Requests (CR) only if no SANs are set.
-The B<subjectName> is also used as fallback for the sender field
-of outgoing CMP messages if no B<cert> and no B<oldcert> are available.
+The I<subjectName> is also used as fallback for the sender field
+of outgoing CMP messages if no references certificate is available.
```

Comment:
Shouldn't it be rather `reference certificate`?

Reply:
Argh, I thought I had fixed this yesterday, but something must have gone wrong.
Now it's definitely in :smile: